Dockerfile

```
@@ -1,6 +1,6 @@
 # Use the official lightweight Node.js 12 image.
 # https://hub.docker.com/_/node
-FROM node:14-slim
+FROM node:18-slim
```

```suggestion
FROM node:18.17.1-bookworm-slim
```

Stricter pin for the package.json `volta.node`.